htmd/parameterization/esp.py

```diff
@@ -298,17 +310,24 @@ def run(self):
 class TestESP(unittest.TestCase):
 
     def setUp(self):
-
         from htmd.home import home
-        from htmd.parameterization.ffmolecule import FFMolecule, FFTypeMethod
+        from htmd.parameterization.fftype import FFTypeMethod, fftype
+        from htmd.parameterization.util import getEquivalentsAndDihedrals, canonicalizeAtomNames
+        from htmd.molecule.molecule import Molecule
 
         molFile = os.path.join(home('test-param'), 'H2O2.mol2')
-        self.mol = FFMolecule(molFile, method=FFTypeMethod.GAFF2)
+        mol = Molecule(molFile)
+        mol = canonicalizeAtomNames(mol)
+        mol, equivalents, all_dihedrals = getEquivalentsAndDihedrals(mol)
```

Comment:
No atom renaming and atom typing needed here

Reply:
Removed